Spinit.CosmosDb/Validation/ThroughputValidator.cs

```diff
@@ -1,7 +1,27 @@
-﻿namespace Spinit.CosmosDb.Validation
+﻿using Microsoft.Azure.Cosmos;
+
+namespace Spinit.CosmosDb.Validation
 {
     internal static class ThroughputValidator
     {
-        internal static bool IsValidThroughput(int throughput) => throughput % 100 == 0 && throughput >= 400 && throughput <= 1000000;
+        internal static bool IsValidManualThroughput(int throughput) => throughput % 100 == 0 && throughput >= 400 && throughput <= 1000000;
+        internal static bool IsValidAutoscaleThroughput(int throughput) => throughput % 1000 == 0 && throughput >= 4000 && throughput <= 1000000;
```

Comment:
Är inte det minsta 400?
(har inte dubbelkollat, men har för mig det..)

Reply:
För fast throughput är 400 minsta ja, men [vid autoscale](https://docs.microsoft.com/en-us/azure/cosmos-db/provision-throughput-autoscale) sätter man max-gränsen och undre gränsen är 10% av det (så 4000 max ger 400 min), om jag förstått det hela rätt.

Reply:
... [fast enligt detta](https://docs.microsoft.com/en-us/azure/cosmos-db/autoscale-faq#what-is-the-entry-point-ru-s-for-autoscale-) så skall man kunna sätta 100-1000, från och med april i år... Kanske borde sänka den till 1000 då i stället för autoscale

Reply:
Ändrade detta, så man kan sätta 1000 vid autoscale (och då få 100-1000)

Reply:
OK!